### PR TITLE
Small config sample fixes

### PR DESCRIPTION
**Pull request by mmattel**
## Description

Small config sample fixes, no content change.

## Related Issue

No Issue related, just found and fixed

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [x] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

**Review by settermjd — APPROVED** [review]

**Comment by mmattel** [comment]
drone: `webui-acceptance-tests` are failing ??

**Comment by mmattel** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by ownclouders** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by phil-davis** [comment]
Weird drone fail https://drone.owncloud.com/owncloud/core/13759/720
I restarted the drone build

**Comment by phil-davis** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by ownclouders** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Review by phil-davis — APPROVED** [review]

File: config/config.apps.sample.php [hunk 1]
```diff
@@ -18,9 +18,12 @@
  *    own line
  *  * add a `@see CONFIG_INDEX` to copy a previously described config option
  *    also to this line
- *  * everything between the ` *\/` and the next `/**` will be treated as the
+ *  * everything between the `*\/` and the next `/**` will be treated as the
  *    config option
  *  * use RST syntax
+ * If you have multiple possible values in the comment section, seperate them with a
```
Comment (davitol) [inline config/config.apps.sample.php]: TYPO in seperate, JFYI. Seen while testing 10.1 beta Changelog

Comment (mmattel) [reply config/config.apps.sample.php]: pls see fix in #34019

**Pull request merged**